exist-core/src/main/java/org/exist/xquery/functions/fn/FunDocumentURI.java

```diff
@@ -37,7 +37,7 @@
  */
 public class FunDocumentURI extends Function {
 
-    private static final FunctionParameterSequenceType FS_PARAM_NODE = optManyParam("value", Type.NODE, "The document node.");
+    private static final FunctionParameterSequenceType FS_PARAM_NODE = optParam("value", Type.NODE, "The document node.");
```

Comment:
same here, this might/will have impact; v7 would then be the 1st opportunity?

Reply:
Yes v7 would be appropriate.